README.md

```diff
@@ -1,2 +1,22 @@
 # hw01
-Repository for first CFSS homework assignment
+Repository for first CFSS homework assignment   
+
+## HELLO MY NAME IS...   
+
+Nicholena Honors
+
+*this stuff is tough*  
+
+**but I'm getting by**
+
+### I have a ton more to learn clearly...  
+
+#For breakfast I had...
+
+* [TEA](http://g3yxi3w953w3xjjaj1xcbuay.wpengine.netdna-cdn.com/wp-content/uploads/2015/01/Dollarphotoclub_68652169-e1422257422160.jpg)
+* [EGGS](http://www.incredibleegg.org/wp-content/themes/incredibleegg/assets/images/facts-left-egg.png)
+
+
+I found the rMarkdown help and followed the cues there. I struggled at first with seperating lines but realized that and extra spacing line would work after just messing around for a bit.
```

Comment:
Yes. That helped me too. I think delving into the R cheatsheets will be helpful in general, once we get a hang of it.